packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -356,6 +364,24 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     return undefined;
   }
 
+  private _getFeedback(): JSX.Element | undefined {
+    if (this.props.feedback) {
+      return (
+        <div className="ComponentPage-feedbackSection">
+          <h2 className="ComponentPage-subHeading ComponentPage-variantsTitle" id="Feedback">
+            Feedback
+          </h2>
+          <a href="https://github.com/OfficeDev/office-ui-fabric-react/issues" target="_blank">
```

Comment:
Please use the Fabric Link component instead of <a>, so the style adheres to Fabric styles as well.